packages/peregrine/lib/talons/MiniCart/miniCart.gql.js

```diff
@@ -0,0 +1,18 @@
+import { gql } from '@apollo/client';
+
+// We disable linting for local fields because there is no way to add them to
+// the fetched schema.
+// https://github.com/apollographql/eslint-plugin-graphql/issues/99
+/* eslint-disable graphql/template-strings */
```

Comment:
Same comment here re: re-enabling this lint rule, although you may be able to just remove `@client`.